### PR TITLE
Add requested configs + more + fixes

### PR DESCRIPTION
**Pull request by Whennya**
Adds configs for:
- ze_cyberspace_v1_s
- ze_dark_souls_ptd_v0_4
- ze_ffvii_malgo_reactor_css4_2
- ze_ffxii_mt_bur_omisace_v6fix
- ze_ffxii_salikawood_v1_8s
- ze_illya_b4
- ze_ouroboros_v3
- ze_rizomata_s2_1
- ze_luciddreams_css4
- ze_eerie_escape_redux_v2
- ze_ffxiv_wanderers_palace_v6css

Fixes for:
- ze_ffvii_malgo_reactor_css4_2
- ze_ffxiv_wanderers_palace_v5_2f
- ze_ffxiv_wanderers_palace_v6
- ze_lotr_isengard_v2_3
- ze_luciddreams_v3_6_f2
- ze_santassination_css2
- ze_xenovation_v3_5_s
- zs_obj_rescape_v19
- ze_ffxiv_wanderers_palace_v5_2f.cfg

### CODE REVIEW

File: musicname/ze_dark_souls_ptd_v0_4.cfg [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"music"
+{
+    "anor_londo_music_1_1.mp3"          "Dark Souls 3 OST - Twin Princes"
+  //"anor_londo_music_2_1.mp3"          ""
```
Comment (notkoen) [inline musicname/ze_dark_souls_ptd_v0_4.cfg]: According to legacy config, `anor_londo_music_2_1.mp3` should be `"Demon's Souls OST - Maiden Astraea (Hellkite Remix)"`. Will need to double check.

File: musicname/ze_eerie_escape_redux_v2.cfg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "biohazard.mp3"                 "Resident Evil OST - Main Title Theme"
+    "ff12dragonrider2.mp3"          "Two Steps From Hell - Dragon Rider"
+    "ffvii_9000miles.mp3"           "Pendulum - 9000 Miles"
+    "ffvii_rageawake2.mp3"          "Kingdom Hearts OST - Fate of the Unknown"
+    "ffvii_tempest.mp3"             "Pendulum - Tempest"
+    "invincible.mp3"                "Two Steps From Hell - Invincible"
+    "level5_end_song (1).mp3"       "Immediate Music - Dark Side Of Power"
+    "stage3_music2.mp3"             "Two Steps From Hell - Protectors of the Earth"
+    "stage4_music1.mp3"             "Pendulum - Crush"
+    "TSFH_Victory.mp3"              "Two Steps From Hell - Victory"
```
Comment (notkoen) [inline musicname/ze_eerie_escape_redux_v2.cfg]: Capital file name

File: musicname/ze_ffxii_salikawood_v1_8s.cfg
Comment (notkoen) [inline musicname/ze_ffxii_salikawood_v1_8s.cfg:1]: Some of the music is from Kingdom (the anime?), follow `<Artist> - <Song Name> (Kingdom OST)` format in this case.

File: musicname/ze_dark_souls_ptd_v0_4.cfg (hunk 1 above)
Comment (Whennya) [reply musicname/ze_dark_souls_ptd_v0_4.cfg]: looks right, didnt check old config

File: musicname/ze_ffxii_salikawood_v1_8s.cfg [hunk 3]
```diff
@@ -0,0 +1,21 @@
+"music"
+{
+    "yamamoto kouta - the one.mp3"                                      "The One - Kingdom OST - KOHTA YAMAMOTO"
+    "yamamoto kouta - set off.mp3"                                      "Set Off - Kingdom OST - KOHTA YAMAMOTO"
+    "yamamoto kouta - hung in a bad place.mp3"                          "Hung in a Bad Place - Kingdom OST - KOHTA YAMAMOTO"
+    "studio eim - pvp.mp3"                                              "StudioEIM - Vindictus PVP Theme 3"
+    "studio eim - elchulus the ancient.mp3"                             "StudioEIM - 고대 엘쿨루스 (Elchulus the Ancient)"
+    "really slow motion & giant apes - vega.mp3"                        "Really Slow Motion & Giant Apes - Vega(Epic Dark Orchestral)"
```
Comment (notkoen) [inline musicname/ze_ffxii_salikawood_v1_8s.cfg:8]: Remove `(Epic Dark Orchestral)` at the end, extra detail

Comment (notkoen) [inline musicname/ze_ffxii_salikawood_v1_8s.cfg]: Remove extra detail at the end
```suggestion
    "really slow motion & giant apes - vega.mp3"                        "Really Slow Motion & Giant Apes - Vega"
```

File: musicname/ze_ffxii_salikawood_v1_8s.cfg [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "nishiki yasunori - octopath traveler.mp3"                          "Octopath Traveler OST - Main Theme"
+    "immediate music - atlas.mp3"                                       "Immediate Music - Atlas"
+    "hitoshi sakimoto - the dalmasca eastersand.mp3"                    "Final Fantasy XII OST - The Dalmasca Eastersand"
```
Comment (notkoen) [inline musicname/ze_ffxii_salikawood_v1_8s.cfg]: Follow conventions we agreed on from before; Use `FFXII OST`

File: musicname/ze_ffxiv_wanderers_palace_v6.cfg
Comment (notkoen) [inline musicname/ze_ffxiv_wanderers_palace_v6.cfg:1]: We can probably remove this file instead. `ze_ffxiv_wanderers_palace_v6` is a csgo version that wont be used. `v6css` is what's on sunrust so better change to this.

File: musicname/ze_ffxiv_wanderers_palace_v5_2f.cfg
Comment (notkoen) [inline musicname/ze_ffxiv_wanderers_palace_v5_2f.cfg:1]: If you don't mind, there are extra music on this version than the csgo version that you should check for.

File: musicname/ze_ouroboros_v3.cfg [hunk 5]
```diff
@@ -0,0 +1,8 @@
+"music"
+{
+    "the caretaker - everywhere at the end of time - 35 f5 - aching cavern without lucidity.mp3"            "The Caretaker - Everywhere at The End of Time - 35 f5 - Aching CAvern without Lucidity"
```
Comment (notkoen) [inline musicname/ze_ouroboros_v3.cfg]: Typo, capital A in `CAvern`
```suggestion
    "the caretaker - everywhere at the end of time - 35 f5 - aching cavern without lucidity.mp3"            "The Caretaker - Everywhere at The End of Time - 35 f5 - Aching Cavern without Lucidity"
```

File: musicname/ze_rizomata_s2_1.cfg
Comment (notkoen) [inline musicname/ze_rizomata_s2_1.cfg:1]: I see `NieR Replican ver.1.22` listed twice, one with and one without the OST. Also, for `5_music2_v2.mpe` the OST information should be in parentheses.
`5_bossmusic2.mp3` says `NieR Gestalt Replicant Original Soundtrack`, can be shortened to just `NieR Gestalt Replicant OST`

File: musicname/ze_rizomata_s2_1.cfg [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    "3_music3.mp3"                  "Mhysa - Game of Thrones - Season 3 - Soundtrack"
+    "3_bossmusic_v2.mp3"            "NieR OST - Gods Bound by Rules"
+    "3_music4.mp3"                  "Finale- Game of Thrones - Season 1 - Soundtrack"
+    "4_music.mp3"                   "Two Steps From Hell - El Dorado (SkyWorld)"
+    "4_music2.mp3"                  "The Island - 14 My Name Is Lincoln"
+    "4_music3.mp3"                  "Becoming One Of The People-Becoming One With Neytiri - James Horner HD"
+    "4_music4.mp3"                  "Two Steps From Hell - Clock Tower Parade"
+    "4_bossmusic_v2.mp3"            "NieR OST. - Emil Karma"
+    "5_music.mp3"                   "Fantasy Music - Age of Wonders"
+    "5_music2_v2.mp3"               "Kainé - Escape ｜ NieR Replicant ver 1.22 OST"
```
Comment (notkoen) [inline musicname/ze_rizomata_s2_1.cfg]:
```suggestion
    "5_music2_v2.mp3"               "Kainé - Escape (NieR Replicant ver 1.22 OST)"
```

File: musicname/ze_rizomata_s2_1.cfg [hunk 7]
```diff
@@ -0,0 +1,22 @@
+"music"
+{
+    "1_music.mp3"                   "Fantasy Celtic Music - Spirit of the Wild"
+    "1_music2.mp3"                  "Fantasy Medieval Music - Rise of a Kingdom"
+    "2_music.mp3"                   "Epic Fantasy Music - Mother Earth"
+    "2_music2_v2.mp3"               "Ori and the Blind Forest - Escaping the Ruins"
+    "2_bossmusic.mp3"               "NieR Replicant ver.1.22 - The Dark Colossus Destroys All (Percussion Mix)"
```
Comment (notkoen) [inline musicname/ze_rizomata_s2_1.cfg]:
```suggestion
    "2_bossmusic.mp3"               "NieR Replicant ver 1.22 - The Dark Colossus Destroys All"
```

File: musicname/ze_rizomata_s2_1.cfg [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+    "3_music.mp3"                   "Ivan Torrent - Human Legacy"
+    "3_music2.mp3"                  "Ramin Djawadi - Main Title[Game of Thrones]"
```
Comment (notkoen) [inline musicname/ze_rizomata_s2_1.cfg]:
```suggestion
    "3_music2.mp3"                  "Ramin Djawadi - Main Title (Game of Thrones OST)"
```

File: musicname/ze_rizomata_s2_1.cfg [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    "3_music3.mp3"                  "Mhysa - Game of Thrones - Season 3 - Soundtrack"
```
Comment (notkoen) [inline musicname/ze_rizomata_s2_1.cfg]:
```suggestion
    "3_music3.mp3"                  "Game of Thrones: Season 3 OST - Mhysa"
```

File: musicname/ze_rizomata_s2_1.cfg [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+    "3_bossmusic_v2.mp3"            "NieR OST - Gods Bound by Rules"
+    "3_music4.mp3"                  "Finale- Game of Thrones - Season 1 - Soundtrack"
+    "4_music.mp3"                   "Two Steps From Hell - El Dorado (SkyWorld)"
```
Comment (notkoen) [inline musicname/ze_rizomata_s2_1.cfg]: Same stuff but I think you get it

File: musicname/ze_ffxii_salikawood_v1_8s.cfg [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+    "yamamoto kouta - the one.mp3"                                      "KOHTA YAMAMOTO - The One"
+    "yamamoto kouta - set off.mp3"                                      "KOHTA YAMAMOTO - Set Off"
+    "yamamoto kouta - hung in a bad place.mp3"                          "KOHTA YAMAMOTO - Hung in a Bad Place"
+    "studio eim - pvp.mp3"                                              "StudioEIM - Vindictus PVP Theme 3"
+    "studio eim - elchulus the ancient.mp3"                             "StudioEIM - 고대 엘쿨루스 (Elchulus the Ancient)"
+    "really slow motion & giant apes - vega.mp3"                        "Really Slow Motion & Giant Apes - Vega"
+    "nishiki yasunori - octopath traveler.mp3"                          "Octopath Traveler OST - Main Theme"
+    "immediate music - atlas.mp3"                                       "Immediate Music - Atlas"
+    "hitoshi sakimoto - the dalmasca eastersand.mp3"                    "FFXII OST - The Dalmasca Eastersand"
+    "hitoshi sakimoto - streets of rabanastre.mp3"                      "FFXII: The Zodiac Age OST Streets of Rabanastre"
```
Comment (notkoen) [inline musicname/ze_ffxii_salikawood_v1_8s.cfg]:
```suggestion
    "hitoshi sakimoto - streets of rabanastre.mp3"                      "FFXII: The Zodiac Age OST - Streets of Rabanastre"
```

File: musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg [hunk 12]
```diff
@@ -32,10 +32,17 @@
...
     // Other
     "pendulum_witchcraft.mp3"                                   "Pendulum - Witchcraft (Live at Glostonbury 2011)"
     "pendulum_ghosts_n_stuff.mp3"                               "deadmau5 ft. Rob Swire - Ghosts 'n Stuff (Pendulum Remix)"
     "sinners_heist_streetlight_people.mp3"                      "Sinners Heist ft. Harley - Streetlight People"
+
+    // Added in v6fix
+    "audiomachine_smoke_screen.mp3"                             "Audiomachine - Smoke Screen"
+    "metrik_time_to_let_go.mp3"                                 "Metrik - Time To Let Go"
+    "phoenix_rising2.mp3"                                       "Edgar Hopp - Phoenix Rising"
+    "zardonic_follow_me2.mp3"                                   "Zardonic - Follow Me"
```
Comment (notkoen) [inline musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg]:
```suggestion
    "zardonic_follow_me2.mp3"                                   "Zardonic - Follow Me (ft. Camo MC)"
```

File: musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg [hunk 13]
```diff
@@ -32,10 +32,17 @@
...
+    "zardonic_takeover.mp3"                                     "Zardonic - Takeover"
```
Comment (notkoen) [inline musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg]:
```suggestion
    "zardonic_takeover.mp3"                                     "Zardonic - Takeover (ft. The Qemists)"
```

File: musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg [hunk 14]
```diff
@@ -32,10 +32,17 @@
     "pain_push_the_pusher_(zardonic_remix).mp3"                 "PAIN - Push the Pusher (Zardonic Remix)"
     "pendulum_archangel.mp3"                                    "Pendulum - Archangel (Vassa Music Cover)"
     "the_qemists_lost_weekend.mp3"                              "The Qemists - Lost Weekend"
-    "zardonic_follow_me.mp3"                                    "Zardonic ft. Camo MC - Follow Me"
+  //"zardonic_follow_me.mp3"                                    "Zardonic ft. Camo MC - Follow Me"
 
     // Other
     "pendulum_witchcraft.mp3"                                   "Pendulum - Witchcraft (Live at Glostonbury 2011)"
     "pendulum_ghosts_n_stuff.mp3"                               "deadmau5 ft. Rob Swire - Ghosts 'n Stuff (Pendulum Remix)"
     "sinners_heist_streetlight_people.mp3"                      "Sinners Heist ft. Harley - Streetlight People"
```
Comment (notkoen) [inline musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg]:
```suggestion
    "sinners_heist_streetlight_people.mp3"                      "Sinners Heist - Streetlight People (ft. Harley)"
```

File: musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg
Comment (notkoen) [inline musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg:1]: `the_twelve_towers.mp3` and `voicians_uncharted.mp3` are deleted from the map based on what Malgo said.

File: musicname/ze_rizomata_s2_1.cfg [hunk 15]
```diff
@@ -0,0 +1,22 @@
...
+    "3_music3.mp3"                  "Game of Thrones: Season 3 OST - Mhysa"
+    "3_bossmusic_v2.mp3"            "NieR Replicant ver.1.22 OST - Gods Bound by Rules"
+    "3_music4.mp3"                  "Game of Thrones: Season 1 OST - Finale"
+    "4_music.mp3"                   "Two Steps From Hell - El Dorado (SkyWorld)"
+    "4_music2.mp3"                  "The Island - 14 My Name Is Lincoln"
+    "4_music3.mp3"                  "James Horner HD - Becoming One Of The People-Becoming One With Neytiri"
+    "4_music4.mp3"                  "Two Steps From Hell - Clock Tower Parade"
+    "4_bossmusic_v2.mp3"            "NieR Replicant ver.1.22 OST - Emil Karma"
+    "5_music.mp3"                   "Fantasy Music - Age of Wonders"
+    "5_music2_v2.mp3"               "NieR Replicant ver 1.22 OST - Kainé / Escape"
```
Comment (notkoen) [inline musicname/ze_rizomata_s2_1.cfg]:
```suggestion
    "5_music2_v2.mp3"               "NieR Replicant ver.1.22 OST - Kainé / Escape"
```

File: musicname/ze_rizomata_s2_1.cfg [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+    "5_bossmusic2.mp3"              "NieR Gestalt Replicant Original Soundtrack. Deep Crimson Foe"
```
Comment (notkoen) [inline musicname/ze_rizomata_s2_1.cfg]:
```suggestion
    "5_bossmusic2.mp3"              "NieR Gestalt Replicant OST - Deep Crimson Foe"
```

**Pull request merged**